Sources/SwiftDocC/SwiftDocC.docc/SwiftDocC/RenderingModel/RenderInlineContent.md

```diff
@@ -0,0 +1,21 @@
+# ``SwiftDocC/RenderInlineContent``
```

Comment:
Nice, this is great improvement! Just wondering if we should place the 'Inline content types' task group first though, because they're more likely to be used? Interested in hearing your thoughts though.

Reply:
Agree. Fixed. 👍